index.html

```diff
@@ -40,6 +41,182 @@ <h2>I'am a Software Developer</h2>
+                            when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <ul class="project-tech mobile-tech">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                    </ul>
+                    <button class="project-button mobile-button">See Project</button>
+                </div>
+            </article>
+        </section>
+        <section class="about-section">
+            <header class="about-header">
+                <h3>About Me</h3>
+            </header>
+            <article class="about-desc">
+                <p class="description mobile-description">
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been 
+                    the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a standard dummy text.
+                </p>
+                <div class="abt-bottom">
+                    <div class="project-button about-btn">My Resume</div>
+                    <div class="abt-pic">
+                        <img alt="about image illustration" src="images/about.png">
+                    </div>
+                </div>
+                <hr>
+            </article>
+            <article class="languages">
+                <header class="lang-header">
+                    <img alt="buttet-icon" src="images/bullet.png">
+                    <h3 class="desc-title lang-title">Languages</h3>
+                </header>
+                <ul class="lang-list">
+                    <li>Javascript</li>
+                    <li>Html</li>
+                    <li>Ruby</li>
+                    <li>Css</li>
+                </ul>
+            </article>
+            <article class="languages">
+                <header class="lang-header">
+                    <img alt="buttet-icon" src="images/bullet.png">
+                    <h3 class="desc-title lang-title">Frameworks</h3>
+                </header>
+                <ul class="lang-list">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </article><article class="languages">
+                <header class="lang-header">
+                    <img alt="buttet-icon" src="images/bullet.png">
+                    <h3 class="desc-title lang-title">Skills</h3>
+                </header>
+                <ul class="lang-list">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </article>
```

Comment:
- [ ] About section is looking great! Let's make it even better by listing the technology categories too using `ul` and `li` tags, as this is a better practice for listing in HTML:
```
<ul>
   <li>Languages</li>
      <ul>
          <li>HTML</li>
          <li>CSS</li>
          <li>JS</li>
       </ul>
    <li>Frameworks</li>
      <ul>
          <li>HTML</li>
          <li>CSS</li>
          <li>JS</li>
       </ul>
       ....
```